File: docs/howto/go/hello.go
package main

import (
	"encoding/json"
	"fmt"
	"os"
)

type Person struct {
	Name string
}

func main() {
	p := &Person{}
	if err := json.NewDecoder(os.Stdin).Decode(p); err != nil {
		fmt.Println("error:", err)
		return
	}
	fmt.Println("Hello go! My name is", p.Name)
}

File: docs/howto/python/hello.py
import sys
sys.path.append("packages")
import fcntl
import os
import json

def non_block_read(output):
    fd = output.fileno()
    fl = fcntl.fcntl(fd, fcntl.F_GETFL)
    fcntl.fcntl(fd, fcntl.F_SETFL, fl | os.O_NONBLOCK)
    try:
        return output.read()
    except:
        return ""

try:
	content = non_block_read(sys.stdin)
	obj = json.loads(content)
	if obj["name"] != "":
		name = obj["name"]
except ValueError:
	name = "World"

print "Hello", name, "!!!"